src/test/evo_deterministicmns_tests.cpp

```diff
@@ -260,10 +259,11 @@ BOOST_FIXTURE_TEST_CASE(dip3_protx, TestChain400Setup)
     // Mine 20 blocks, checking MN reward payments
     std::map<uint256, int> mapPayments;
     for (size_t i = 0; i < 20; i++) {
+        SyncWithValidationInterfaceQueue();
         auto dmnExpectedPayee = deterministicMNManager->GetListAtChainTip().GetMNPayee();
+        SyncWithValidationInterfaceQueue();
```

Comment:
This second `SyncWithValidationInterfaceQueue` isn't needed.

Reply:
Yep. removed.